charts/agent-stack-k8s/templates/_helper.tpl

```diff
@@ -0,0 +1,6 @@
+{{/* Generate basic labels */}}
+{{- define "agent-stack-k8s.labels" }}
+  labels:  
+    {{- toYaml $.Values.labels }}
+    app: {{ .Release.Name }}
```

Comment:
```suggestion
    {{- toYaml $.Values.labels }}
    app: {{ .Release.Name }}
```

Reply:
Otherwise, `labels` appears twice and there will be another error:
```
Error: YAML parse error on agent-stack-k8s/templates/deployment.yaml.tpl: error converting YAML to JSON: yaml: line 14: mapping values are not allowed in this context
```

This is what the template renders as:
```
# Source: agent-stack-k8s/templates/deployment.yaml.tpl
apiVersion: apps/v1
kind: Deployment
metadata:
  name: agent-stack-k8s-clusters
  namespace: buildkite-clusters
spec:
  selector:
    matchLabels:
      app: agent-stack-k8s-clusters
  template:
    metadata:
      labels:
          labels:user: user
            app: agent-stack-k8s
```
if you run it with `--set labels.user=user`. Notice that the indentation is also not consistent. I will repeat my recommendation that you DO NOT do any indentation in the helper, and use `[nindent](https://helm.sh/docs/chart_template_guide/function_list/#nindent)` (not `indent`) in the yaml file to control the indentation.

I recommend you attempt to render the template with `help template --debug` during your development, so avoid having to go back and forth like this.

Reply:
Thank you. I am not very familiar with helm cmd.  `help template --debug` command did not work in my git repo or I missed some pre-requisition step 
```
help template --debug 
bash: help: no help topics match `--debug'.  Try `help help' or `man -k --debug' or `info --debug'
```

Reply:
Sorry, that was a typo, its `helm`, not `help`.